opal/mca/mca.h

```diff
@@ -315,9 +329,13 @@ struct mca_base_component_2_1_0_t {
   mca_base_register_component_params_2_0_0_fn_t mca_register_component_params;
   /**< Method for registering the component's MCA parameters */
 
+  int32_t mca_component_flags;
+  /**< flags for this component */
+
   /** Extra space to allow for expansion in the future without
-      breaking older components. */
-  char reserved[32];
+      breaking older components.
+      NTH/JJH: reduced by 4 bytes for mca_component_flags */
```

Comment:
This comment should likely either be explained further or moved to the commit message.  I.e., if you look at this comment 12 months from now, will it still be relevant?  Or stale?  Do we need to know in the code that the padding was reduced by 4?  Or only when spelunking through old commits?  (I'm guessing the latter)